RedJanvier/eshuri-bank-fe
# feat(card search) Implement user interface of card search

**What does this PR do?**
Implements Card Search and Change PIN page.

Description of Task to be completed?
Implements the frontend UI design of Card Search page where user can Search Card and Change Pin.

How should this be manually tested?
After npm run dev , open the /card.html URL in browser.
You should see a form for account creation.

**Note:**
Backend APIs for this page is not ready.
It's Still in progress.
![Card Search](https://github.com/user-attachments/assets/e2a4141a-d700-4d03-ba7c-58a32a19274c)
